app/routes/git_tokens.py

```
@@ -4,340 +4,354 @@
+		git_label = await GitLabel.create(
+			label=payload.label,
+			user_id=payload.user_id,
+			git_hosting=payload.git_hosting,
+			token_value=encrypted_token,
+			username=user.get("username", ""),
+		)
+		
+		return APIResponse.success(
+			message=constants.TOKEN_SAVED_SUCCESSFULLY, data={"id": str(git_label.id)}
+		)
+	except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)